public/js/home.js

```diff
@@ -131,5 +131,23 @@ $.get('/conversation/search', {
 				$contactsContent.append('<span class="recent-name"><a href="#user-modal" class="recent-contact" data-id="'+ uniqueNames[i].dataset.id +'" rel="leanModal">'+ uniqueNames[i].innerHTML +'</a></span>');
 			}  
 		}
+
+		// Displaying convoshapes
+		var $convoEntry = $('.convo');
+		$convoEntry.each(function(){
+			var $numOfNames = $(this).children('.name').length;
+			if ($numOfNames === 2){
+				var newSvg = '<svg id="participantImage" version="1.1" id="Layer_1" xmlns="http://www.w3.org/2000/svg" xmlns:xlink="http://www.w3.org/1999/xlink" x="0px" y="0px" width="100px" height="100px" viewBox="0 0 100 100" enable-background="new 0 0 100 100" xml:space="preserve"> <line id="line1" fill="none" stroke="#555555" stroke-width="2" stroke-miterlimit="10" x1="20.822" y1="20.822" x2="79.18" y2="79.179"/> <circle id="circle1" cx="20.823" cy="20.822" r="8.394"/> <circle id="circle2" cx="79.179" cy="79.18" r="8.394"/> </svg>';
```

Comment:
Another small nit: it might be more clear to group up these `var` declarations into one and place it `var newSvg;` outside the if statements.